lib/improver/tests/weighted_blend/test_blending_utilities.py

```diff
@@ -0,0 +1,264 @@
+    21.11200487, 20.99792316, 22.07955526, 21.71937432,
+    23.12910858, 22.94415916, 25.11682828, 25.76481447,
+    23.82016292, 22.11742526, 26.30864878, 23.81492893])
+
+BLENDED_PERCENTILE_DATA1 = np.array([
+    12.308173106352402, 14.610489348891615,
+    15.914782442039872, 15.108875220912573,
+    17.737443276652936, 17.097492097934627,
+    19.274275011968893, 19.127165271387128,
+    18.742074248084133, 18.50663225732617,
+    20.584984938812934, 20.560776453138736,
+    20.25126805925584, 19.769776454949138,
+    21.64008410795689, 21.598099315407907,
+    22.17361605670607, 21.085160674433627,
+    22.51323580222509, 23.192691941719808,
+    23.82016292415821, 22.944159163666185,
+    26.30864878110861, 25.76481446912205])
+
+BLENDED_PERCENTILE_DATA2 = np.array([
+    12.30817311, 14.61048935,
+    15.91478244, 15.10887522,
+    17.676775957972925, 17.593947064474417,
+    19.51651034470578, 19.378940557706372,
+    18.819297828987274, 18.594620794715546,
+    20.965096170815123, 20.95201851550872,
+    20.890749057567294, 19.843891344879836,
+    21.927325497880343, 22.32506953618696,
+    22.491628272758373, 21.143852884603188,
+    22.546838489123576, 23.730102965015696,
+    23.82016292, 22.94415916,
+    26.30864878, 25.76481447])
+
+PERCENTILE_VALUES = np.array(
+    [[12.70237152, 14.83664335, 16.23242317, 17.42014139, 18.42036664,
+      19.10276753, 19.61048008, 20.27459352, 20.886425, 21.41928051,
+      22.60297787],
+     [17.4934137, 20.56739689, 20.96798405, 21.4865958, 21.53586395,
+      21.55643557, 22.31650746, 23.26993755, 23.62817599, 23.6783294,
+      24.64542338],
+     [16.24727652, 17.57784376, 17.9637658, 18.52589225, 18.99357526,
+      20.50915582, 21.82791334, 21.90645982, 21.95860878, 23.52203933,
+      23.71409191]])
+
+
+def percentile_cube():
+    """Create a percentile cube for testing."""
+    data = np.reshape(PERCENTILE_DATA, (6, 2, 2, 2))
+    cube = Cube(data, standard_name="air_temperature",
+                units="C")
+    cube.add_dim_coord(DimCoord(np.linspace(-45.0, 45.0, 2), 'latitude',
+                                units='degrees'), 2)
+    cube.add_dim_coord(DimCoord(np.linspace(120, 180, 2), 'longitude',
+                                units='degrees'), 3)
+    time_origin = "hours since 1970-01-01 00:00:00"
+    calendar = "gregorian"
+    tunit = Unit(time_origin, calendar)
+    cube.add_dim_coord(DimCoord([402192.5, 402193.5],
+                                "time", units=tunit), 1)
+    cube.add_dim_coord(DimCoord([0, 20, 40, 60, 80, 100],
+                                long_name="percentile_over_realization"), 0)
+    return cube
+
+
+class Test__repr__(IrisTest):
+
+    """Test the repr method."""
+
+    def test_basic(self):
+        """Test that the __repr__ returns the expected string."""
+        result = str(BlendingUtilities())
+        msg = '<BlendingUtilities>'
+        self.assertEqual(result, msg)
+
+
+class TestBlendingUtilities(IrisTest):
```

Comment:
Fixed with basic docstrings, which will need refining when we review the unit tests